assets/js/components/notifications/ZeroDataStateNotifications/GatheringDataNotification.js

```diff
@@ -35,7 +35,10 @@ import GatheringDataIcon from '../../../../svg/graphics/zero-state-red.svg';
 import { getTimeInSeconds, trackEvent } from '../../../util';
 import useViewContext from '../../../hooks/useViewContext';
 
-export default function GatheringDataNotification( { title } ) {
+export default function GatheringDataNotification( {
+	title,
+	gatheringDataWaitTime,
```

Comment:
```suggestion
	gatheringDataWaitTime,
```

It's a bit odd that we've left this as a string, and also that we aren't using `_n()` for translations. I know in this instance we don't expect non-plural amounts, but we should be consistent with our translation approaches and use:

1. numbers not strings for numerical values
2. `_n()` to account for different values in different locales

I know @10upsimon is out today so I'll make this minor change then get this merged.